src/client.ts

```diff
@@ -309,25 +310,26 @@ export class PluggyClient extends BaseApi {
 
   /**
    * Creates a Webhook
-   * @param event The type of event to listen
-   * @param url The url where will receive notifications
-   * @returns {Webhook} a webhook object
+   * @param webhookParams - The webhook params to create, this includes:
+   * - url: The url where will receive notifications
+   * - event: The event to listen for
+   * - headers (optional): The headers to send with the webhook
+   * @returns {Webhook} the created webhook object
    */
-  async createWebhook(event: WebhookEvent, url: string): Promise<Webhook> {
+  async createWebhook(webhookParams: CreateWebhook): Promise<Webhook> {
     return this.createPostRequest(`webhooks`, null, {
-      event,
-      url,
+      ...webhookParams,
```

Comment:
I don't understand, why are we changing the signature of this method, at all?

Reply:
We were only giving part of the config to the user to create a webhook, instead of adding more parameters, we merged it into a single object to have add all the possibles parameters instead of `url` and `event`

Reply:
this is a breaking change and completely unjustified for the change itself. 
1- it's not incorrect to add more parameters if they are ortogonal/unrelated to the previous one.
2- this new parameter is not mandatory so it could simply have been added as optional.
3- we aren't adding any new or critical behavior to justify a breaking change or major release.

Please rollback this change and keep the previous API.